include/numerics/parsed_function_parameter.h

```diff
@@ -24,11 +24,12 @@
 // Local Includes -----------------------------------
 #include "libmesh/libmesh_common.h"
 #include "libmesh/parameter_accessor.h"
-#include "libmesh/parsed_function.h"
 
 namespace libMesh
 {
 
+template <typename T> class ParsedFunction;
```

Comment:
> On Nov 4, 2015, at 5:56 PM, Paul T. Bauman <notifications@github.com> wrote:
> 
> In include/numerics/parsed_function_parameter.h:
> 
> >  
> >  namespace libMesh
> >  {
> >  
> > +template <typename T> class ParsedFunction;
> This forward declaration is incorrect. ParsedFunction has two template parameters (maybe cppclean missed it because we're always using the default second argument within this class?). I think we need to keep the header here, for now, and remove this forward declaration. @roystgnr, do we always want the default second template argument for ParsedFunction in ParsedFunctionParameter?
> 

Hmm, good catch, no idea why that compiled...